src/jvmMain/kotlin/com/sdercolin/vlabeler/ui/editor/ChartStore.kt

```diff
@@ -139,6 +146,15 @@ class ChartStore {
                         )
                     }
                 }
+                if (sampleInfo.hasFundamental && sampleInfo.hasSpectrogram && appConf.painter.fundamental.enabled) {
```

Comment:
Same here. `sampleInfo.hasSpectrogram` is redundant?

Reply:
Remove `sampleInfo.hasSpectrogram`.